src/ontology/mondo-edit.obo

```diff
@@ -2640,11 +2640,9 @@ replaced_by: MONDO:0017572
 
 [Term]
 id: MONDO:0000275
-name: monogenic disease
-def: "A genetic disease that is the result of one or more abnormal alleles and may be dominant (a single copy of the abnormal allele is sufficient to give rise to the disease), semi-dominant, or recessive (requiring both copies of the gene to have an abnormal allele)." [DO:wk\,ls, DOID:0050177]
-comment: Editor notes: consider obsoleting
-xref: DOID:0050177 {source="MONDO:equivalentTo"}
-is_a: MONDO:0003847 {source="DOID:0050177"} ! inherited genetic disease
+name: obsolete monogenic disease
+is_obsolete: true
+replaced_by: MONDO:0003847
```

Comment:
I think this should be consider. not all inherited diseases are monogenic. E.g. digenic.

Reply:
@mellybelly requested this and suggested the replacement term. Is there a better term to replace 'monogenic disease'?

Reply:
or do we want to keep it?

Reply:
@cmungall should we keep 'monogenic disease'? I can close this PR if yes

Reply:
logically monogenic is a subtype of inherited, as is digenic, etc. We may decide we don't have sufficient information to classify under mono/di, in which case obsoletion is justified. However, my point is that replaced_by should only be used with an equivalent term. Here it would be consider.

Reply:
Ah - I get it now! Thanks for the clarification. :) I just made that change.